content/en/docs/ops/configuration/multi-revision/gateways/index.md

```diff
@@ -0,0 +1,216 @@
+---
```

Comment:
I think this doc would be better located in the setup section. It should also have links to and from the upgrade doc.

Maybe the upgrade doc should be split into two docs (canary and in-place) under a new upgrade folder and this doc could then be a third doc under the upgrade folder?